pom.xml

```diff
@@ -11,6 +11,11 @@
   <version>4.0.0.Beta2-SNAPSHOT</version>
   <packaging>pom</packaging>
   <name>Quarkus - Operator SDK - Parent</name>
+  <properties>
+    <quarkus.version>2.8.2.Final</quarkus.version>
+    <java-operator-sdk.version>3.0.0.RC3</java-operator-sdk.version>
+    <kubernetes-client.version>5.12.2</kubernetes-client.version>
+  </properties>
```

Comment:
Not sure about this change… This make it easier to deal from our side but wouldn't that make it harder to consume the BOM from a user's perspective?

Reply:
I confirm it still works